ui/src/AppCharting/AppChartC3.svelte

```diff
@@ -155,25 +157,16 @@
       return;
 
     const selectedSet = new Set(selectedTimestamps);
-
     const bySize = chartType === BY_SIZE;
 
     const xLabels = chartData.columns[0].slice(1);
 
-    const active = [...errors.values()].filter(error_list => {
-      const {scenario: es, key} = error_list[0];
+    const active = [...errors.values()].filter(error => {
+      if (!error.length) return false;
```

Comment:
It looks like this function doesn't return anything. If that's true then `return false` should be just `return`.

Reply:
this is a fitler function.